requirements.txt

```diff
@@ -19,3 +19,4 @@ tox==2.3.1
 uwsgi==2.0.12
 Pillow==3.1.1
 wagtail==1.4.3
+-e git+git://github.com/mitodl/edx-api-client.git#egg=edx-api-client
```

Comment:
we are still too unstable with the library: as soon as I add the certificates part, I will make a release and change this